src/components/WebexSettings/Tabs.jsx

```diff
@@ -18,22 +19,52 @@ import webexComponentClasses from '../helpers';
  * @param {string} props.selected  Specifies which tab is active
  * @param {object} props.style  Custom style to apply
  * @param {Tab[]} props.tabs  The tabs to be rendered
+ * @param props.tabIndex
  * @returns {object} JSX of the component
- *
  */
 export default function Tabs({
   className,
   onSelect,
   selected,
   style,
   tabs,
+  tabIndex,
 }) {
   const {content} = tabs.find((tab) => tab.key === selected);
   const cssClasses = webexComponentClasses('tabs', className);
+  const tabListRef = useRef();
+  const leftPress = useKeyPress('ArrowLeft', tabListRef);
+  const rightPress = useKeyPress('ArrowRight', tabListRef);
+
+  useEffect(() => {
+    const selectedTabIndex = tabs.findIndex((tab) => tab.key === selected);
+
+    if (tabs && leftPress) {
```

Comment:
a little late to check if `tabs` is defined, `tabs.findIndex` would already throw an error. You could just use:
```js
(tabs || []).findIndex(...);
```